pkg/server/sandbox_run.go

```diff
@@ -199,7 +200,11 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 	glog.V(5).Infof("Create sandbox container (id=%q, name=%q).",
 		id, name)
 	// We don't need stdio for sandbox container.
```

Comment:
Move this comment down before `containerd.NullIO`.

Reply:
Done.